packages/terra-form-input/src/Input.scss

```diff
@@ -0,0 +1,92 @@
+:local {
+  .form-input {
+    background-color: var(--terra-form-input-background-color, #fff);
+    background-image: var(--terra-form-input-focus-animation-background, none);
+    background-position: 0 100%;
+    background-repeat: no-repeat;
+    background-size: var(--terra-form-input-focus-background-start, 0 0);
+    border: var(--terra-form-input-border, 1px solid #4a4a4a);
+    border-radius: var(--terra-form-input-border-radius, 0.2rem);
+    box-shadow: var(--terra-form-input-box-shadow, none);
+    color: var(--terra-form-input-color, #000);
+    font-family: inherit;
+    font-size: var(--terra-form-input-font-size, 1.143rem);
+    line-height: var(--terra-form-input-line-height, 1.143);
+    opacity: var(--terra-form-input-opacity, 1);
+    padding-bottom: var(--terra-form-input-padding-bottom, 0.571rem);
+    padding-left: var(--terra-form-input-padding-left, 0.357rem);
+    padding-right: var(--terra-form-input-padding-right, 0.357rem);
+    padding-top: var(--terra-form-input-padding-top, 0.571rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border: var(--terra-form-input-hover-border, 1px solid #4a4a4a);
+    }
+
+    &[required] {
+      box-shadow: unset;
```

Comment:
For browser compatibility, it may be better to set this to `none`. Unset doesn't work in IE browsers or most of the Android browsers. https://caniuse.com/#search=unset

Reply:
This came directly from [here](https://github.com/cerner/terra-core/blob/master/packages/terra-form-textarea/src/Textarea.scss#L35). I've made the change here 840073f2a7864a6bc4ee4d9c10501b5903ba460f